### PR TITLE
fixed FPR deployment script

### PR DESCRIPTION
**Pull request by STYJ**

### CODE REVIEW

File: web3deployment/reserveDeployer.js [hunk 1]
```diff
@@ -98,26 +98,109 @@ async function deployContract(solcOutput, contractName, ctorArgs) {
...
-  "KyberNetwork.sol" : fs.readFileSync(contractPath + 'KyberNetwork.sol', 'utf8'),
-  "WhiteList.sol" : fs.readFileSync(contractPath + 'WhiteList.sol', 'utf8'),
-  "KyberReserveInterface.sol" : fs.readFileSync(contractPath + 'KyberReserveInterface.sol', 'utf8'),
-  "Withdrawable.sol" : fs.readFileSync(contractPath + 'Withdrawable.sol', 'utf8'),
-  "KyberReserve.sol" : fs.readFileSync(contractPath + 'KyberReserve.sol', 'utf8'),
-  "Wrapper.sol" : fs.readFileSync(contractPath + 'mockContracts/Wrapper.sol', 'utf8')
-};
+  'ConversionRatesInterface.sol': fs.readFileSync(
+    contractPath + 'ConversionRatesInterface.sol',
+    'utf8'
```
Comment (ilanDoron) [inline web3deployment/reserveDeployer.js]: please use the compile contracts file from branch development.
Can add it to this deploy.
if its lacking any contract please add it.

File: web3deployment/reserveDeployer.js [hunk 2]
```diff
@@ -153,9 +236,9 @@ function parseInput( jsonInput ) {
 
       tokens.push(address);
       const dict = {
-        minimalRecordResolution : web3.utils.toBN(val["minimalRecordResolution"]),
-        maxPerBlockImbalance : web3.utils.toBN(val["maxPerBlockImbalance"]),
-        maxTotalImbalance : web3.utils.toBN(val["maxTotalImbalance"])
+        minimalRecordResolution : val["minimalRecordResolution"],
+        maxPerBlockImbalance : val["maxPerBlockImbalance"],
+        maxTotalImbalance : val["maxTotalImbalance"]
       };
```
Comment (ilanDoron) [inline web3deployment/reserveDeployer.js]: I don't follow, why don't we need BN here anymore?
or it was never needed?

Comment (STYJ) [reply web3deployment/reserveDeployer.js]: The web3.utils.toBn function returns a BN object. Passing this into the function will also fail and return an error like `<rejected> Error: invalid number value (arg="minimalRecordResolution", coderType="uint256", value="38d7ea4c68000")`

The values stored in the config file (ropsten.json) is already a string, there is no need to convert it to BN.

File: web3deployment/reserveDeployer.js (hunk 1 above)
Comment (STYJ) [reply web3deployment/reserveDeployer.js]: are we referring to this? https://github.com/KyberNetwork/smart-contracts/blob/development/compilation.sh

File: web3deployment/reserveDeployer.js (hunk 2 above)
Comment (STYJ) [reply web3deployment/reserveDeployer.js]: As for the package-lock, I wasn't sure why it changed when I did npm-install. Anyway, I will remove it.

File: web3deployment/reserveDeployer.js (hunk 1 above)
Comment (ilanDoron) [reply web3deployment/reserveDeployer.js]: this one:
https://github.com/KyberNetwork/smart-contracts/blob/development/web3deployment/compileContracts.js

File: web3deployment/reserveDeployer.js (hunk 2 above)
Comment (ilanDoron) [reply web3deployment/reserveDeployer.js]: yes please remove it.
we have started using npm ci for a new repo clone
and not npm i.

anyway package lock shouldn't be added to PRs unless packages changed as well.

Comment (STYJ) [reply web3deployment/reserveDeployer.js]: Understood. Thanks for letting me know!

**Pull request merged**